FightPandemics/Features/Feed/CreatePost/CreatePostBody.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreatePostBody.swift
+//  FightPandemics
+//
+//  Created by ehsan sat on 5/13/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class CreatePostBody: UIView {
+    private var title = UITextField()
+    private var body = UITextView()
+    func setUp () {
+        // set Up Title
+        title.placeholder = "Title"
+        title.attributedPlaceholder = NSAttributedString(string: "Title", attributes: [NSAttributedString.Key.font: Fonts.poppinsBold.customFont(size: 22), NSAttributedString.Key.foregroundColor: UIColor(hexString: "#939393")])
+        title.backgroundColor = UIColor.clear
+        title.layer.backgroundColor = UIColor.clear.cgColor
+        title.layer.borderWidth = 0
+        title.layer.borderColor = UIColor.clear.cgColor
+        self.addSubview(title)
+        // set up body
+        body.text = "Description of your post."
```

Comment:
Description of your post

Reply:
and again, make sure to localize all user-facing strings